catkin_tools/context.py

```diff
@@ -71,7 +71,7 @@ class Context(object):
         'space_suffix']
 
     @classmethod
-    def Load(cls, workspace_hint=None, profile=None, opts=None, strict=False):
+    def Load(cls, workspace_hint=None, profile=None, opts=None, strict=False, append=False):
```

Comment:
nitpick: I guess I missed this on my original review, but methods should always by lowercase: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables